internal/kubernetes/operator/features/crds.go

```diff
@@ -106,6 +107,25 @@ var (
 			resource{ResourceAtlasNetworkContainer, NopPatcher()},
 			resource{ResourceAtlasNetworkPeering, NopPatcher()},
 		},
+		"2.9.0": {
```

Comment:
Please, remove 2.6.0, we only keep the last 3 majors.

Reply:
actually I am fixing in this one instead, you are right the 2.6 deprecation needs to happen already.